gui/tslint.json

```diff
@@ -37,6 +37,7 @@
     "max-classes-per-file": [
       false,
       1
-    ]
+    ],
+    "curly": [true, "ignore-same-line"]
```

Comment:
Can we remove the "ignore-same-line" flag? I feel the code to be more readable when we add curly braces for control statements.

Reply:
This is now a very standard practice of not adding curly braces when you have only one line code for control statement.

Reply:
Okay